deployment/ansible/group_vars/development

```diff
@@ -15,3 +15,7 @@ postgresql_host: "{{ services_ip }}"
 relp_host: "{{ services_ip }}"
 graphite_host: "{{ services_ip }}"
 statsite_host: "{{ services_ip }}"
+
+sqs_queue: "raster-foundry-{{ host_user }}-{{ host_hostname }}"
```

Comment:
Just in case the same user wanted to run the app on different machines and keep things separated.